src/Tribe/Rewrite.php

```diff
@@ -389,6 +403,30 @@ protected function add_hooks() {
 		add_action( 'tribe_events_pre_rewrite', array( $this, 'generate_core_rules' ) );
 		add_filter( 'post_type_link', array( $this, 'filter_post_type_link' ), 15, 2 );
 		add_filter( 'url_to_postid', array( $this, 'filter_url_to_postid' ) );
+		add_action( 'wp_loaded', [ $this, 'maybe_delayed_flush_rewrite_rules' ] );
+	}
+
+	/**
+	 * When dealing with flush of rewrite rules we cannot do it from the activation process due to not all classes being
+	 * loaded just yet. We flag a transient without expiration on activation so that on the next page load we flush the
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 119 characters

<sub>hash: b4ba1ee29cac77effc56dd6b85918ba5</sub>